### PR TITLE
BookStore : Connect To API

### PR DESCRIPTION
**Pull request by bhushan354**
In this PR I have implemented following Features : 

- Made use of the [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) as my backend.
- Used [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) to make HTTP requests to the backend.
- Fetched books from the Bookstore API after the components has rendered.
- Refactored add book and remove book features to persist your changes in the server.
- Made sure that application should behave the same way after the refactoring.

Thank You ,
Bhushan .

### CODE REVIEW

**Comment by bhushan354** [comment]
Hello sir , I hope my submission passes your review but if there are any small changes required,  as There is time constraints so it would be of great help to me if you contact me on slack or whatsapp (+91 8600118932) about it .Thank you for consideration

Thank You ,
Bhushan .

File: src/redux/books/bookSlice.js [hunk 1]
```diff
@@ -1,40 +1,83 @@
-import { createSlice } from '@reduxjs/toolkit';
+/* eslint-disable import/no-extraneous-dependencies */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
```
Comment (TadesseDev) [inline src/redux/books/bookSlice.js]: - [ ] Please Consider removing disabled liters. Remember linter is here to force you to write clean code and help you identify what could possibly be improved on the current codebase.

**Comment by TadesseDev** [comment]
@bhushan354, I tried to reach you on slack but you were not responding.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @bhushan354 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] The app is functioning as expected ✅ 
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Brushan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job implementing the requested changes.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**